packages/@react-aria/dialog/docs/useDialog.mdx

```diff
@@ -44,8 +44,7 @@ building fully accessible custom dialogs from scratch is very difficult and erro
 `useDialog`, combined with <TypeLink links={overlaysDocs.links} type={overlaysDocs.exports.useModal} />,
 helps achieve accessible dialogs that can be styled as needed.
 
-**Note: `useDialog` only handles the dialog itself. It should be combined with other hooks and
-components as described below to create a fully accessible modal dialog.**
+**Note:** `useModal` only hides content within parent `OverlayProvider` components. However, if you have additional content in your application outside any `OverlayProvider`, then you should use the `@react-aria/aria-modal-polyfill` package to ensure that this content is hidden while modals are open as well. See the [aria-modal-polyfill docs](../react-aria/aria-modal-polyfill.html) for more information.
```

Comment:
`parent 'OverlayProvider' components` reads weird to me. I'm wondering if you should remove parent or add child.

Reply:
This is hard to describe but I think both Devon and I had a go at explaining this and what Rob has feels like the best of the bunch.